tests/conftest.py

```diff
@@ -787,6 +787,31 @@ def design_execution_dict(generic_entity):
     return ret
 
 
+@pytest.fixture
+def generative_design_execution_dict(generic_entity):
+    ret = generic_entity.copy()
+    return ret
+
+
+@pytest.fixture
+def example_generation_results():
+    return {
+        "page": 2,
+        "per_page": 4,
+        "response": [{
+            "id": str(uuid.uuid4()),
+            "execution_id": str(uuid.uuid4()),
+            "result": {
+                "seed": str,
```

Comment:
Found your test problem. These need to be values, not classes.

Reply:
OMG lol. :D